### PR TITLE
Support readonly in TinyMCE and ACE editor

### PR DESCRIPTION
**Pull request by zonky2**
Add readonly at tinyMCE and ACE editor if set in DCA

### CODE REVIEW

File: core-bundle/src/Resources/contao/classes/DataContainer.php [hunk 1]
```diff
@@ -697,6 +697,7 @@ protected function row($strPalette=null)
 			$objTemplate->type = $type;
 			$objTemplate->fileBrowserTypes = $fileBrowserTypes;
 			$objTemplate->source = $this->strTable . '.' . $this->intId;
+			$objTemplate->readonly = (bool) $arrData['eval']['readonly'];
```
Comment (ausi) [inline core-bundle/src/Resources/contao/classes/DataContainer.php]:
```suggestion
			$objTemplate->readonly = (bool) ($arrData['eval']['readonly'] ?? false);
```

**Review by aschempp — DISMISSED** [review]

File: core-bundle/src/Resources/contao/templates/backend/be_ace.html5 [hunk 2]
```diff
@@ -47,6 +47,9 @@ window.ace && window.addEvent('domready', function() {
   // Disable command conflicts with AltGr (see #5792)
   editor.commands.bindKey('Ctrl-alt-a|Ctrl-alt-e|Ctrl-alt-h|Ctrl-alt-l|Ctrl-alt-s', null);
 
+  // Add readonly if set in DCA
+  <?php if ($this->readonly): ?>editor.setOptions({readOnly, true,});<?php endif; ?>
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/templates/backend/be_ace.html5]: This looks weird to me. Did you mean `{readOnly: true}`?

**Review by leofeyer — CHANGES_REQUESTED** [review]

File: core-bundle/src/Resources/contao/templates/backend/be_tinyMCE.html5 [hunk 3]
```diff
@@ -76,7 +76,8 @@ window.tinymce && tinymce.init({
 
   browser_spellcheck: true,
   tabfocus_elements: ':prev,:next',
-  importcss_append: true
+  importcss_append: true,
+  <?php if ($this->readonly): ?>readonly: 1,<?php endif; ?>
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/templates/backend/be_tinyMCE.html5]: According to the documentation, possible values are `true` and `false` but not `1`.



See https://www.tiny.cloud/docs/tinymce/6/editor-important-options/#starting-the-editor-in-a-read-only-state

**Review by leofeyer — CHANGES_REQUESTED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @zonky2.

**Pull request merged**